src/watchkit.cs

```diff
@@ -1791,4 +1791,133 @@ interface WKInterfaceTextField {
 		[Export ("setSecureTextEntry:")]
 		void SetSecureTextEntry (bool secureTextEntry);
 	}
+
+	[Watch (7,0), NoiOS]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface WKApplication
+	{
+		[Static]
+		[Export ("sharedApplication")]
+		WKApplication SharedApplication { get; }
+
+		[Export ("openSystemURL:")]
+		void OpenSystemUrl (NSUrl url);
+
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
+		IWKApplicationDelegate Delegate { get; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; }
+
+		[NullAllowed, Export ("rootInterfaceController")]
+		WKInterfaceController RootInterfaceController { get; }
+
+		[NullAllowed, Export ("visibleInterfaceController")]
+		WKInterfaceController VisibleInterfaceController { get; }
+
+		[Export ("applicationState")]
+		WKApplicationState ApplicationState { get; }
+
+		[Export ("isApplicationRunningInDock")]
+		bool IsApplicationRunningInDock { get; }
+
+		[Export ("autorotating")]
+		bool Autorotating { [Bind ("isAutorotating")] get; set; }
```

Comment:
I'm fine with this, we tend to drop the `Is` prefix, but not sure if `Autorotated`is the best name. Though I can't think of anything better beyond adding Is, and then we're Inconsistent.

Reply:
I'm with you, I think in this case it reads better with an is. So lets do it, we will be like French, more exceptions that rules ;)